src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Random random = new Random();
+        String color = ColorSupplier.getRandomColor();

```

Comment:
You should create private final ColorSupplier colorSupplier = new ColorSupplier();
and here get color colorSupplier.getRandomColor()